src/test/java/au/org/emii/MyIT.java

```diff
@@ -11,8 +11,11 @@
 import java.util.Properties;
```

Comment:
I'm going to investigate publishing this project using https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+Maven+Repository+Server from jenkins - so that the netcdf service can get it more easily...  so don't worry about jenkins for now.